dev/com.ibm.ws.wssecurity.fat.utils.common/fat/src/com/ibm/ws/wssecurity/fat/utils/common/CommonTests.java

```diff
@@ -699,7 +699,7 @@ private static void setupSSLClient() {
 
         /*
```

Comment:
Maybe this commented out code should just be removed?

Reply:
Removed.